lte/gateway/python/magma/mobilityd/main.py

```diff
@@ -13,6 +13,7 @@
 import ipaddress
 import logging
 from typing import Any, Optional
+import sdnotify
```

Comment:
Just for my understanding: is the reason for only giving mobilityd and pipelined these notifiers because they are the only python services that take some time to restart?

Reply:
The notifier is added to sctpd and sessiond as well. These services are all restarted together with the mme https://github.com/magma/magma/blob/351a724e32a9cbffed184ba660152fe3290696a3/lte/gateway/deploy/roles/magma/files/systemd_bazel/magma%40mme.service#L13